docs/UserGuide.md

```diff
@@ -568,7 +575,7 @@ Listed here are some frequently asked questions. Should you require any other as
 --------------------------------------------------------------------------------------------------------------------
 
 ## Contact Us
-For enquiries or feedback, feel free to let us know at [guestbook.enquiries@gmail.com](mailto:guestbook.enquiries@gmail.com).
+For enquiries or feedback, feel free to let us know at [guestbook.enquiries@gmail.com](mailto:guestbook.enquiries@gmail.com), and we will get back to you within 3 working days!
```

Comment:
Are we going to ensure this AHAH

Reply:
Hmm if not we can just say as soon as possible, I think will be good enough!